core/src/main/scala/Workflow.scala

```diff
@@ -162,6 +162,7 @@ object Workflow {
 
     def writePKIRoleToVault(dc: Datacenter, sn: StackName): WorkflowF[Unit] =
       Vault.createPKIRole(
+        pkiPath = dc.policy.pkiPath,
```

Comment:
I'm not 100% sure this is what you wanted. Right here this is going to give you the literal value including the `%` template values. I think what you in fact want would be something that utilizes some of the functions here: https://github.com/getnelson/nelson/blob/master/core/src/main/scala/vault/policies.scala#L27 - you may also want to consider writing a function for this particular interpolation, as it looks like you might need a path that is ever so slightly different to the ones being used in the policy?

Reply:
OH okay, that makes more sense - I was under the impression that this was something that was configured before-Nelson 😅 will take a second look~

Reply:
Yeah so it is configured prior to bootup, but because of the need to template these paths, they have to be interpolated at runtime. The config only holds the templatized values 🥾